client/devicemonitor/client.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package devicemonitor
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/rest_utils"
+	"github.com/pkg/errors"
+)
+
+const (
+	AlertsURI = "/api/v1/devicemonitor/tenants/:tenant_id/devices/:device_id/alerts/latest"
+	HealthURI = "/api/v1/health"
+)
+
+const (
+	defaultTimeout = time.Duration(5) * time.Second
+)
+
+// Client is the workflows client
```

Comment:
is this comment copy-paste ro do we mean it?

Reply:
it is; fixed